treeherder/webapp/api/runnable_jobs.py

```diff
@@ -14,6 +17,21 @@ def list(self, request, project):
         """
         GET method implementation for list of all runnable buildbot jobs
         """
+        if 'decisionTaskID' in request.query_params and len(request.query_params['decisionTaskID']):
+            decisionTaskID = request.query_params['decisionTaskID']
+            tc_jobs_url = "https://queue.taskcluster.net/v1/task/" + decisionTaskID + "/artifacts/public/full-task-graph.json"
+            tc_graph = None
+            validate = URLValidator()
+            try:
+                validate(tc_jobs_url)
+                tc_graph = fetch_json(tc_jobs_url)
+            except ValidationError:
+                pass
+            except Exception as ex:
+                return Response("Exception: {0}".format(ex), 500)
```

Comment:
`Response()` only takes one positional argument, the second argument is actually a named argument, called `status`. We've also just switched to using the rest framework constants (see bug 1279880), rather than ints - for example:
```python
    return Response("Exception: {0}".format(ex), status=HTTP_500_INTERNAL_SERVER_ERROR)
```
(You'll need to `from rest_framework.status import HTTP_500_INTERNAL_SERVER_ERROR`)